python/tests/io/test_pyarrow.py

```diff
@@ -377,7 +378,7 @@ def test_timestamp_type_to_pyarrow() -> None:
 
 def test_timestamptz_type_to_pyarrow() -> None:
     iceberg_type = TimestamptzType()
-    assert visit(iceberg_type, _ConvertToArrowSchema()) == pa.timestamp(unit="us", tz="+00:00")
+    assert visit(iceberg_type, _ConvertToArrowSchema()) == pa.timestamp(unit="us", tz="UTC")
```

Comment:
Why are these changes needed?

Reply:
I'm not sure, it just popped up at some point. Reverted it

Reply:
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   pyarrow.lib.ArrowInvalid: Schema at index 4 was different: 
E   VendorID: int64
E   tpep_pickup_datetime: timestamp[us, tz=UTC]
E   tpep_dropoff_datetime: timestamp[us, tz=UTC]
E   passenger_count: double
E   trip_distance: double
E   RatecodeID: double
E   store_and_fwd_flag: string
E   PULocationID: int64
E   DOLocationID: int64
E   payment_type: int64
E   fare_amount: double
E   extra: double
E   mta_tax: double
E   tip_amount: double
E   tolls_amount: double
E   improvement_surcharge: double
E   total_amount: double
E   congestion_surcharge: double
E   airport_fee: double
E   vs
E   VendorID: int64
E   tpep_pickup_datetime: timestamp[us, tz=+00:00]
E   tpep_dropoff_datetime: timestamp[us, tz=+00:00]
E   passenger_count: double
E   trip_distance: double
E   RatecodeID: double
E   store_and_fwd_flag: string
E   PULocationID: int64
E   DOLocationID: int64
E   payment_type: int64
E   fare_amount: double
E   extra: double
E   mta_tax: double
E   tip_amount: double
E   tolls_amount: double
E   improvement_surcharge: double
E   total_amount: double
E   congestion_surcharge: double
E   airport_fee: double

pyarrow/error.pxi:100: ArrowInvalid
```

It looks like that Arrow gives preference to UTC over +0:00 when creating an empty dataframe.

Reply:
Fixed in https://github.com/apache/iceberg/pull/6946. Closing this thread.